src/transformers/chai-assert.js

```diff
@@ -0,0 +1,310 @@
+            j.callExpression(j.identifier('expect'), [actual]),
+            j.identifier(identifier)
+        ), Array.isArray(expectation) ? expectation : [expectation]);
+
+    const makeNegativeExpectation = (identifier, actual, expectation = []) => j.callExpression(j.memberExpression(
+            j.memberExpression(
+                j.callExpression(j.identifier('expect'), [actual]),
+                j.identifier('not')
+            ),
+            j.identifier(identifier)
+        ), Array.isArray(expectation) ? expectation : [expectation]);
+
+    assertToExpectMapping.forEach(({ assert, expect, ignoreExpectedValue, includeNegative, expectedOverride }) => {
+        let override;
+        if (expectedOverride) {
+            override = typeof expectedOverride === 'boolean' ? j.literal(expectedOverride) : j.identifier(expectedOverride);
+        }
+
+        ast.find(j.CallExpression, getAssertionExpression(assert))
+            .replaceWith(path => makeExpectation(expect, ...getArguments(path, ignoreExpectedValue, override)));
+
+        if (includeNegative) {
+            ast.find(j.CallExpression, getAssertionExpression(includeNegative))
+                .replaceWith(path => makeNegativeExpectation(expect, ...getArguments(path, ignoreExpectedValue, override)));
+        }
+    });
+
+    unsupportedAssertions.forEach(assertion => {
+        ast.find(j.CallExpression, getAssertionExpression(assertion)).forEach(path => {
+            logWarning(`Unsupported Chai Assertion "${assertion}" found at line ${path.value.loc.start.line}`);
+        });
+    });
+
+    ['approximately', 'closeTo'].forEach(assertion => {
+        ast.find(j.CallExpression, getAssertionExpression(assertion))
+            .replaceWith(path => makeExpectation('toBeCloseTo', path.value.arguments[0], path.value.arguments.slice(1, 3)));
+    });
+
+    // assert.fail -> expect(false).toBeTruthy()
+    ast.find(j.CallExpression, getAssertionExpression('fail'))
+        .replaceWith(path => makeExpectation('toBe', j.literal(false), j.literal(true)));
+
+    // assert.propertyVal -> expect(*.[prop]).toBe()
+    ast.find(j.CallExpression, getAssertionExpression('propertyVal')).replaceWith(path => {
+        const [obj, prop, value] = path.value.arguments;
+        return makeExpectation('toBe', j.memberExpression(obj, prop), value);
+    });
+
+    // assert.propertyNotVal -> expect(*.[prop]).not.toBe()
+    ast.find(j.CallExpression, getAssertionExpression('propertyNotVal')).replaceWith(path => {
+        const [obj, prop, value] = path.value.arguments;
+        return makeNegativeExpectation('toBe', j.memberExpression(obj, prop), value);
+    });
+
+    // assert.property -> expect(prop in obj).toBeTruthy()
+    ast.find(j.CallExpression, getAssertionExpression('property'))
+        .replaceWith(path => makeExpectation('toBeTruthy', j.binaryExpression('in', path.value.arguments[1], path.value.arguments[0])));
```

Comment:
Sorry for nitpicking here. I see I started by setting `max-len` to 200 in the eslint configuration, but might be nice to break the long lines here to ease readability